bin/luigi

```diff
@@ -1,6 +1,14 @@
 #!/usr/bin/env python
 
-import luigi
+import sys
+import warnings
+import luigi.cmdline
+
+
+def main(argv):
+    warnings.warn("'bin/luigi' has moved to console script 'luigi'", DeprecationWarning)
```

Comment:
@gpoulin, is there any documentation for me how to run this if `bin/luigi` is deprecated?